guides/common/attributes-base.adoc

```diff
@@ -47,6 +47,7 @@
 :customrpmtitle: RPM
 :customssl: SSL
 :customssltitle: SSL
+:firewalld-profile: RH-Satellite-6
```

Comment:
there is a `foreman` profile that can be used for non-Katello/non-Satellite installs:

```
  <include service="dns"/>
  <include service="http"/>
  <include service="https"/>
  <include service="dhcp"/>
  <include service="tftp"/>
  <port protocol="udp" port="68"/>
  <port protocol="tcp" port="8140"/>
```

Reply:
Yesterday I looked at this too, but didn't want to dive too deep into it (rabbit holes ahead).

> <port protocol="udp" port="68"/>

I really do wonder why this was specified. It includes the dhcp service and that's why we need to open it, right?

And it would be nice if there was a firewalld service for Puppet instead of listing the port.

Reply:
Mysteries, all over the place.

https://github.com/firewalld/firewalld/pull/645 points at https://bugzilla.redhat.com/show_bug.cgi?id=1839781 which never was touched by anyone from the Foreman/Satellite team :(